app/src/main/java/bg/zahov/app/ui/history/HistoryFragment.kt

```diff
@@ -32,8 +31,7 @@ class HistoryFragment : Fragment() {
             setContent {
                 HistoryScreen(historyViewModel) {
                     findNavController().navigate(
-                        R.id.history_to_history_info,
-                        bundleOf(WORKOUT_ID_ARG_KEY to it)
+                        R.id.history_to_history_info
```

Comment:
removed the usage of sending ids through arguments to replace it with the stateflow inside workoutProvider